compiler/test/compilable/test12268.d

```diff
@@ -0,0 +1,6 @@
+/* REQUIRED_ARGS: -betterC
```

Comment:
Typo in the issue number in the file name, it should be called `test19268.d`

Reply:
fixed